README.md

```diff
@@ -93,6 +93,29 @@ Velocidi.match("someProvider", listOf(UserId("<Advertising ID>", "gaid"), UserId
 
 For more information about Cross-Channel Matches go to https://docs.velocidi.com/collect/matches/ .
 
+### Collecting IDs
+
+Velocidi's SDK does not collect any user ID by default. These IDs have to be explicitly provided when executing the requests.
+We recommend taking a look at Android's documentation about [Best practices for unique identifiers](https://developer.android.com/training/articles/user-data-ids).
+
+#### Using your own first-party IDs
+
+You can use any ID in our list of [supported IDs](https://docs.velocidi.com/collect/user-ids/#default-id-types) with
+[`com.velocidi.UserId`](https://github.com/velocidi/velocidi-android-sdk/blob/master/velocidi-sdk/src/main/kotlin/com/velocidi/UserId.kt).
+If you would like to use an ID not present in our list of supported IDs, please contact our support so that we can add it.
+
+#### Using the GAID
+
+Google Advertising ID (GAID) is a user-resettable identifier that uiquely identifies a particular user for advertising use cases.
+
+This ID can be fetched using Android's [Adversiting ID library](https://developer.android.com/jetpack/androidx/releases/ads).
```

Comment:
Should we point to https://developer.android.com/reference/androidx/ads/identifier/package-summary instead of the release notes?

Reply:
Changed in 1d766e7